packages/@aws-cdk/aws-lambda/lib/function-base.ts

```diff
@@ -210,7 +210,7 @@ export abstract class FunctionBase extends Resource implements IFunction {
    */
   public addPermission(id: string, permission: Permission) {
     if (!this.canCreatePermissions) {
-      // FIXME: Report metadata
+      // FIXME: @deprecate this behavior in V2; throw an error if calling `addPermission` on a resource that doesn't support it.
```

Comment:
use `@deprecated` as that would that be the search keyword

```suggestion
      // FIXME: @deprecated(v2) - throw an error if calling `addPermission` on a resource that doesn't support it.
```